packages/tools/devtools/devtools-core/src/FluidDevtools.ts

```diff
@@ -525,13 +507,11 @@ export function tryGetFluidDevtools(): IFluidDevtools | undefined {
 }
 
 /**
- * Converts a {@link PureDataObject} into a {@link DecomposedIContainer} by wrapping its underlying {@link IFluidDataStoreRuntime}.
+ * Converts a {@link PureDataObject} into a {@link DecomposedContainer} by wrapping its underlying {@link IFluidDataStoreRuntime}.
  * This allows data objects to be registered with devtools by exposing their runtime properties and events
- * through a {@link IContainer}-compatible interface.
+ * through a container-compatible interface.
  */
-export function toDecomposedIContainer(dataObject: PureDataObject): DecomposedIContainer {
+export function toDecomposedContainer(dataObject: PureDataObject): DecomposedContainer {
```

Comment:
Nit: do we need to export this? We only use this in `registerDataObject`, right? Should we just inline this into that method?